### DISCUSSION TITLE
Hexo - Documentação em pt-br

### DISCUSSION THREAD

**Discussion by lucianobarauna** [frontendbr/forum#850] [Dicas]
Fala galera, quero compartilhar com vocês a documentação do [HEXO](https://hexo.io/pt-br/) um gerador de site estático feito em NODE.js.

O Hexo é uma ferramenta simples, rápida e poderosa para criação de um blog ou para gerar sites estáticos. Você escreve postagens em Markdown (ou outras linguagens) e ele gera arquivos HTML com um lindo tema em segundos.

Para quem já ouviu falar sobre geradores de sites estáticos, ele é bem parecido com o [JEKYLL](https://jekyllrb.com/).

Fizemos com muito amor e carinho para trazer esse conteúdo para nossa comunidade e divulgar mais essa ferramenta.

Nosso repositório com os arquivos é esse aqui: [dochexo pt-br](https://github.com/lucianobarauna/dochexo) e caso veja alguma coisa de errado entra em contato comigo **@lucianobarauna** ou o **@ramonsantosdev** no telegram.

Para quem quiser contribuir ou procurar algum conteúdo traduzido tem um site bem legal que cura essas traduções da nossa comunidade, se liga: [DevTranslate](https://devtranslate.github.io/)

Espero que gostem e nosso pull request foi aceito hoje  ;-)

Tamo junto nessa comunidade BR

**Comment by larismourullo** [comment]
Brigada por mencionar o @devtranslate 😍
E @lucianobarauna continue assim com o grande trabalho que tem feito pela comunidade!

PS: Galera bora apoiar as traduções, é de grande importância pra comunidade.
